cms/djangoapps/contentstore/views/course.py

```diff
@@ -346,6 +346,28 @@ def edevate_reindex_course(request, course_key_string):
     with modulestore().bulk_operations(course_key):
         try:
             CoursewareSearchIndexer.do_course_reindex(modulestore(), course_key)
+            CourseAboutSearchIndexer.remove_deleted_items(course_key)
+        except SearchIndexingError as search_err:
+            return HttpResponse(dump_js_escaped_json({
+                "user_message": search_err.error_list
+            }), content_type=content_type, status=400)
```

Comment:
You know you can use `JsonResponse` since Django 1.7? It will also accept dictionary as a first parameter and serialize it automatically :)